src/lib/leaflet-utils/map-factory.js

```diff
@@ -4,6 +4,7 @@ import '@/lib/leaflet-hack'
 import mapConfig from '@/map.config'
 import { EPSG_3857 } from '../../lib/leaflet-utils/projections'
 import createCrs from '../../lib/leaflet-utils/create-crs'
+import northIcon from '../../img/north-arrow-2.svg'
```

Comment:
Is north-arrow  also used?

Reply:
Could you add this to the img src directly? Why do you import it in js?

Reply:
-  changed the image name
- This way we keep things consistent (we import svg's in other files) and leave it up to webpack how to handle the import. This way the SVG can also be optimized and/or be presented as a base64 url